config/x86/kernel-config

```diff
@@ -23,7 +23,7 @@ CONFIG_INIT_ENV_ARG_LIMIT=32
 # CONFIG_COMPILE_TEST is not set
 CONFIG_LOCALVERSION="-burmilla"
 # CONFIG_LOCALVERSION_AUTO is not set
-CONFIG_BUILD_SALT=""
+CONFIG_BUILD_SALT="5.10.4-amd64"
 CONFIG_HAVE_KERNEL_GZIP=y
```

Comment:
It is better to leave this empty on burmilla https://github.com/torvalds/linux/blob/master/init/Kconfig#L171-L178

Reply:
So no way of debugging stuff there?

Reply:
At least those should not be static because it is very useful to be able to build latest 5.10.x patch versions without need to update kernel config.

So if we want to include this setting it should be set by build scripts.

Reply:
I just see people posting kernel OOPSes without any context to get a hand on or to provide a fix without some info - but as you wish.